pgctl/service.py

```diff
@@ -4,19 +4,19 @@
 from __future__ import unicode_literals
 
 import os
+from collections import namedtuple
 from subprocess import check_call
 
 from cached_property import cached_property
 
 
-class Service(object):
-
-    def __init__(self, path, pgctl_app):
-        self.path = path
-        self._pgctl_app = pgctl_app
+class Service(namedtuple('Service', ['path', 'scratch_dir'])):
 
     def __repr__(self):
-        return "Service('{path}')".format(path=self.path.strpath)
+        return "Service(path='{path}', scratch_dir='{scratch_dir}')".format(
+            path=self.path.strpath,
+            scratch_dir=self.scratch_dir.strpath,
+        )
```

Comment:
true except that `self.path` and `self.scratch_dir` are py.path objects

so the repr would be:
Service(path=local(u'/tmp/magic-service'), scratch_dir=local(u'/tmp/magic-service-scratch'))

if you're ok with that (I am) i can drop the __repr__